spec/menu_spec.rb

```diff
@@ -0,0 +1,15 @@
+require './docs/menu.rb' 
+
+describe 'Menu' do
+    it 'should return a hash with 4 item inside' do  
+        menu = Menu.new
+        expect(menu.list).to eq({ 'Latte' => 2.90, 'Cappuccino' => 2.80, 'Cake' => 3.50, 'Sandwich' => 4.00}) 
+    end 
+
+    describe 'list_menu' do
+        it 'should return alist of strings' do
+            menu = Menu.new 
+            expect(menu.list_menu.class).to eq(String)
```

Comment:
Here you could include one of the strings you expect to receive. This would just confirm that you are definately getting back what you expect. expect(menu.list_menu).to include("Item Latte Cost 2.90")